Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/serverexplore/AddNewClusterCtrlProvider.java

```diff
@@ -51,6 +52,10 @@
             "Please click'OK',then accept the untrusted certificate \r\n"+
             "if you want to link to this cluster.Or you can update the \r\n" +
             "host to link to a different cluster.";
+    private static final String BrokenCertificateErrorMsg =
+            "The cluster certificate is broken.\r\n" +
+            "If you want to bypass certificate verification,\r\n" +
+            "please check the option in Tools | Azure | Experimental | Disable SSL Certificate Verification.";
```

Comment:
Suggest to add comment where these menu items defined in case future changes break consistency.